package.json

```diff
@@ -74,6 +75,7 @@
     "jest-environment-jsdom-sixteen": "^1.0.3",
     "less": "^3.11.1",
     "less-loader": "^5.0.0",
+    "lodash.groupby": "^4.6.0",
```

Comment:
Use the `lodash` package directly instead. It's more likely to already be installed by other deps, and we get less duplicate code (`lodash/groupBy` + `lodash.groupby`).
If you're worried about bundle size, then `import ... from 'lodash/...'` should be used instead.